src/javascript/core/user.js

```diff
@@ -8,6 +8,7 @@ let store;
 const defaultUserConfig = {
 	storage: 'cookie',
 	name: 'spoor-id',
+	nameOverride: 'spoor-id',
 	value: null,
 	domain: (document.URL.match(/^https?\:\/\/([^\/?#]+)(?:[\/?#]|$)/i)[1].indexOf('ft.com') > -1 ? 'ft.com' : null)
```

Comment:
This could also use `String.prototype.includes` instead of `.indexOf` as it is included in the default polyfill bundle -- https://polyfill.io/v2/docs/features/#String_prototype_includes